from error_result import ErrorResult
from successful_result import SuccessfulResult